README.md

```diff
@@ -2,87 +2,204 @@
+You can also set the API keys as environment variables.
+This is highly recommended for any production deployment.
+
+* `MINDEE_RECEIPT_API_KEY`
+* `MINDEE_INVOICE_API_KEY`
+* `MINDEE_PASSPORT_API_KEY`
 
+### Parsing Documents
 
-## Parsing methods
+#### Document types
+The document type must be specified when calling the `Client.parse` method.
 
+The object containing the parsed data will be an attribute of the response object.
+The name of this attribute will be the same as the `document_type`
+specified when calling the `Client.parse` method.
+
+Receipts
+```python
+api_response = mindee_client.parse_from_path("/path/to/receipt.jpg", "receipt")
+print(api_response.receipt)
+```
+Invoices
 ```python
-# Call the receipt parsing API and create a receipt object under parsed_data.receipt
-parsed_data = mindee_client.parse_receipt("/path/to/file")
+api_response = mindee_client.parse_from_path("/path/to/invoice.pdf", "invoice")
+print(api_response.invoice)
+```
+Passports
+```python
+api_response = mindee_client.parse_from_path("/path/to/passport.jpg", "passport")
+print(api_response.passport)
+```
 
-# Call the invoice parsing API and create an invoice object under parsed_data.invoice
-parsed_data = mindee_client.parse_invoice("/path/to/file")
+Mixed data flow of invoices and receipts.\
+**Note:** You'll need an API key for _both_ invoice and receipts endpoints.
+```python
+api_response = mindee_client.parse_from_path("/path/to/receipt.jpg", "financial_document")
+print(api_response.financial_document)
+api_response = mindee_client.parse_from_path("/path/to/invoice.pdf", "financial_document")
+print(api_response.financial_document)
+```
 
-# If you have a mixed data flow of invoice and receipt, use financial_document class
-# Call the invoice or receipt parsing API according to your input data type
-# and create a FinancialDocument object under parsed_data.financial_document
-parsed_data = mindee_client.parse_financial_document("/path/to/file")
+### Document Sources
 
-# Call the passport parsing API and create a Passport object under parsed_data.passport
-parsed_data = mindee_client.parse_passport("/path/to/file")
+You can pass your document in three ways.
 
-# Call the license_plates parsing API and create a CarPlate object under parsed_data.license_plate
-parsed_data = mindee_client.parse_license_plate("/path/to/file")
+As a file path:
+```python
+receipt_data = mindee_client.parse_from_path('/path/to/invoice.pdf', "invoice")
 ```
 
-## Input data
-
-You can pass your input file in three ways:
+As a file object:
+```python
+with open('/path/to/receipt.jpg', 'rb') as fp:
+     receipt_data = mindee_client.parse_from_file(fp, "receipt")
+```
 
-From file path
+As a base64 encoded string:
 ```python
-receipt_data = mindee_client.parse_receipt('/path/to/file', input_type="path")
+b64_string = "/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAgGBgcGBQgHBwcJCQgKDBQNDAsLD...."
+receipt_data = mindee_client.parse_from_b64string(b64_string, "receipt")
```

Comment:
I tested this line and it doesn'tt work because the filename arg is missing.

Reply:
yup, it's fixed now.